### ISSUE TITLE
Change schema issues

### ISSUE THREAD

**Issue by monique2208** [bids-standard/bids-validator#123]
It seems with the new validator some of the defined issue codes have changed. We previously used [this](https://github.com/bids-standard/bids-validator/blob/main/src/issues/list.ts) list to find the issue codes, but it seems some of them are not on the list anymore and others have changed. Some of these we have been using to upgrade warnings that we find critical to errors (for example, missing events files). 

I think a more extensive documentation of these codes is planned (#6), but do the changes in this list reflect changes to the validator? And if not, is there a more complete list somewhere that we can use in the meantime?

**Comment by Remi-Gau** [comment]
@effigies 
we got a list of errors and warning in the schema, no?
could we render this somewhere and point the doc to it?

**Comment by effigies** [comment]
Sort of... We have a list of issues without implementations in the schema, but there are many that are one-off inside checks that would need to be aggregated. There are also issues that are defined in the validator:

https://github.com/bids-standard/bids-validator/blob/9e8da1437b28bf9a69cce3a63727570ec719a754/src/issues/list.ts#L4-L207

I would like to autogenerate, which is why this has stalled a little bit.

I think in the short term, we should probably just make a partial list and encourage users to add to it. If it's easier, we could make it a wiki and then periodically pull in updates to the rendered doc.

**Comment by mateuszpawlik** [comment]
Do I understand it correctly that the validator can be configured with the issue codes from both [validator issues](https://github.com/bids-standard/bids-validator/blob/main/src/issues/list.ts) and [schema rules](https://github.com/bids-standard/bids-specification/tree/master/src/schema/rules)?

**Comment by effigies** [comment]
Yes. As much as possible, we've tried to encode issue information into the schema itself, for consistency in case of multiple validators. Some issues are only encountered by the validator. Some may be in the validator for now, but will be moved into the schema as more of BIDS is schematized.

**Comment by mateuszpawlik** [comment]
Thanks @effigies. I understand that. It is only not clear to me which issue codes can be used in the validator configuration. We're observing some weird behavior, but we need to investigate more.

**Comment by monique2208** [comment]
I think we are mostly using the right codes again. At least they seem to be working with the latest version of the validator and I can also find them, either in the list or in the schema. 

We previously used "CUSTOM_COLUMN_WITHOUT_DESCRIPTION" but this was replaced with "TSV_ADDITIONAL_COLUMNS_UNDEFINED". What is a bit confusing to me is that there is also "TSV_ADDITIONAL_COLUMNS_MUST_DEFINE", I am not sure what the difference is but we have never encountered this error.

We also used to upgrade the ages over 89 to an error, but this does not seem to be working. The [the code is in the schema](https://github.com/bids-standard/bids-specification/blob/c16ab8d571af68d46651fbffb710daa35c269cf5/src/schema/rules/checks/privacy.yaml#L41) but I cannot trigger the warning. Could it be that some the schema checks are not implemented?
